.github/ISSUE_TEMPLATE/2-account-issue.yml

```
@@ -0,0 +1,67 @@
+name: '👤 Account issue'
+labels: ['triage']
+description: An issue with your Jenkins account
+
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Before hitting the button, please answer these questions.
+        It's helpful to search the existing GitHub issues first.
+        There is a chance that another user has already reported the issue you're facing, or it's a known issue that we're already aware of. 
+
+        Fill in as much of the template below as you can.
+        The more information we have the better we can help you.
```

```suggestion
        The more information we have, the better we can help you.
```
small formatting suggestion